apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -370,6 +370,26 @@
                             "minimum" : 0
                         }
                     }
+                },
+                "protocol_beneficiary_address" : {
+                    "description" :
+                    "Public key of beneficiary account that belongs to protocol maintainers.",
+                    "type" : "string",
+                    "example" : "ak_DummyPubKeyDoNotEverUse999999999999999999999999999",
+                    "pattern": "^ak_[1-9A-HJ-NP-Za-km-z]*$"
+                },
+                "protocol_beneficiary_enabled" : {
+                    "description" :
+                    "If true, the node will split rewards and send part to protocol_beneficiary_address",
+                    "type" : "boolean"
+                },
+                "protocol_beneficiary_factor" : {
+                    "description" :
+                    "Part of rewards to split and send to protocol_beneficiary_address. Rounded to avoid fractions. 100 is 10%",
```

Comment:
```suggestion
                    "Part of rewards to split and send to protocol_beneficiary_address. Truncated to avoid fractions. 100 is 10%",
```

Examples I tried:
```
1> 123 * 100 div 1000.
12
2> 1234 * 100 div 1000.
123
3> 1239 * 100 div 1000.
123
```